### PR TITLE
[WIP] Network Proxy GRPC Test

### PR DESCRIPTION
**Pull request by Jefftree**
Do not merge. Ensure new network proxy image fixes all e2e flakes.

/test pull-kubernetes-e2e-gce-network-proxy

### CODE REVIEW

File: cluster/gce/addons/konnectivity-agent/daemonset.yaml [hunk 1]
```diff
@@ -47,13 +42,6 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
           resources:
-            limits:
+            requests:
```
Comment (caesarxuchao) [inline cluster/gce/addons/konnectivity-agent/daemonset.yaml]: Do we need merge this change?

Comment (Jefftree) [reply cluster/gce/addons/konnectivity-agent/daemonset.yaml]: I didn't notice any differences to the test passing rate suggesting that there's a bigger problem we need to tackle first.

**Comment by Jefftree** [comment]
Testing is completed.

**Pull request closed without merging**